spec.html

```diff
@@ -22624,15 +22645,8 @@ <h1>ExecuteModule ( ) Concrete Method</h1>
 
           <emu-alg>
             1. Let _module_ be this Source Text Module Record.
-            1. Let _moduleCxt_ be a new ECMAScript code execution context.
```

Comment:
The 'declaration' for ExecuteModule() says that it initializes the execution context of the module, so that wouldn't be true any more.